src/ceph-volume/ceph_volume/devices/raw/activate.py

```diff
@@ -4,14 +4,15 @@
 import os
```

Comment:
@leseb I think all modifications in this file should be removed given we don't need the activation part, right?

Reply:
Yup